.github/settings.yml

```
@@ -0,0 +1,100 @@
+# https://github.com/apps/settings
+
+repository:
```

What about the name?

```suggestion
repository:
  name: docs
```

---

I think I am going to keep it as generic as possible to try to have the same settings for all repositories

---

and what about the `description` and `homepage`?

---

Yes, I am going to remove them too